build-scripts/problem-matchers/clang.json

```diff
@@ -0,0 +1,18 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "cata-clang",
+            "pattern": [
+                {
+                    "regexp": "^(.+):(\\d+):(\\d+):\\s(error|warning):\\s(.+)\\s\\[(.+)\\]$",
+                    "file": 1,
+                    "line": 2,
+                    "column": 3,
+                    "severity": 4,
+                    "message": 5,
+                    "code": 6
+                }
+            ]
+        }
+    ]
+}
```

Comment:
You might find this problem matcher just works:
https://github.com/marketplace/actions/gcc-problem-matcher

Otherwise you might want to publish the problem matcher JSON as an action on its own (like the GCC one) so others can benefit too.

Reply:
I'm mildly in favour of using the existing action, let's go that way unless you have an objection jbtw.

Reply:
OK.  I'll close this and open another PR.